docs/guides/dynamo_run.md

```diff
@@ -155,57 +162,52 @@ cd target/debug
 ```
 > Note: Build with `--release` for a smaller binary and better performance, but longer build times. The binary will be in `target/release`.
 
-To build for other engines, see the following sections.
+### mistralrs
 
-
-### sglang
-
-1. Setup the python virtual env:
+[mistral.rs](https://github.com/EricLBuehler/mistral.rs) is a pure Rust engine that is fast to run, fast to load, supports GGUF as well as safetensors, and runs well on CPU as well as GPU. For those reasons it is the default engine.
 
 ```
-uv venv
-source .venv/bin/activate
-uv pip install pip
-uv pip install sgl-kernel --force-reinstall --no-deps
-uv pip install "sglang[all]==0.4.2" --find-links https://flashinfer.ai/whl/cu124/torch2.4/flashinfer/
+dynamo-run Qwen/Qwen2.5-3B-Instruct
 ```
 
-2. Build
+is equivalent to
 
 ```
-cargo build --features sglang
+dynamo-run in=text out=mistralrs Qwen/Qwen2.5-3B-Instruct
 ```
 
-3. Run
-
-Any example above using `out=sglang` will work, but our sglang backend is also multi-gpu and multi-node.
+### llama_cpp
 
-**Node 1:**
 ```
-cd target/debug
-./dynamo-run in=http out=sglang --model-path ~/llm_models/DeepSeek-R1-Distill-Llama-70B/ --tensor-parallel-size 8 --num-nodes 2 --node-rank 0 --leader-addr 10.217.98.122:9876
+dynamo-run out=llamacpp ~/llms/Llama-3.2-3B-Instruct-Q6_K.gguf
 ```
 
-**Node 2:**
+### sglang
+
+1. Setup the python virtual env:
+
 ```
-cd target/debug
-./dynamo-run in=none out=sglang --model-path ~/llm_models/DeepSeek-R1-Distill-Llama-70B/ --tensor-parallel-size 8 --num-nodes 2 --node-rank 1 --leader-addr 10.217.98.122:9876
+uv venv
+source .venv/bin/activate
+uv pip install pip
+uv pip install sgl-kernel --force-reinstall --no-deps
+uv pip install "sglang[all]==0.4.2" --find-links https://flashinfer.ai/whl/cu124/torch2.4/flashinfer/
 ```
 
-To pass extra arguments to the sglang engine see *Extra engine arguments* below.
+2. Run
 
-### llama_cpp
+Any example above using `out=sglang` will work, but our sglang backend is also multi-gpu.
```

Comment:
Does multi-node support still work well?

Reply:
I don't know. I was able to test tensor parallel on an 8xL40S box yesterday. Today I'll try to find a second one.

Reply:
Tested, multi-node works well.